### PR TITLE
Add additional convenience packages as deps

### PR DESCRIPTION
**Pull request by erelson**
Simplifies installation.

### CODE REVIEW

**Comment by erelson** [comment]
@moriarty I think this is what we need.

File: fetch_system_config/root/etc/netplan/99-fetch-ethernet.yaml.default [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+network:
+  version: 2
+  ethernets:
+    #eth0:
+    #  addresses: [172.42.42.1/24]
+    #  gateway4: 172.42.42.1
+    eth1:
+      addresses: [10.42.42.1/24]
+      gateway4: 10.42.42.1
+EOF
```
Comment (moriarty) [inline fetch_system_config/root/etc/netplan/99-fetch-ethernet.yaml.default]: is EOF still needed? I think that's left over from the `cat << EOF`
```suggestion

```

File: fetch_bringup/package.xml [hunk 2]
```diff
@@ -17,10 +17,13 @@
   <exec_depend>diagnostic_aggregator</exec_depend>
   <exec_depend>fetch_description</exec_depend>
   <exec_depend>fetch_drivers</exec_depend>
+  <exec_depend>fetch_navigation</exec_depend>
+  <exec_depend>fetch_open_auto_dock</exec_depend>
   <exec_depend>fetch_teleop</exec_depend>
   <exec_depend>graft</exec_depend>
   <exec_depend>image_proc</exec_depend>
   <exec_depend>joy</exec_depend>
+  <exec_depend>moveit_ros</exec_depend>
```
Comment (moriarty) [inline fetch_bringup/package.xml]:
```suggestion
  <exec_depend>fetch_moveit_config</exec_depend>
```

I think we probably depend on `fetch_moveit_config`

https://github.com/fetchrobotics/fetch_ros/blob/melodic-devel/fetch_moveit_config/package.xml

Comment (moriarty) [inline fetch_bringup/package.xml]:
```suggestion
  <exec_depend>fetch_moveit_config</exec_depend>
```

**Review by moriarty — CHANGES_REQUESTED** [review]

**Review by moriarty — APPROVED** [review]
LGTM

**Pull request merged**